pythonx/UltiSnips/snippet_manager.py

```diff
@@ -572,6 +574,11 @@ def _try_expand(self):
         if not before:
             return False
         snippets = self._snips(before, False)
+        if snippets:
```

Comment:
Because it is useful for having non-context snippets as default snippet with same trigger as context snippets.

Like this:
```
snippet i "default if clause" b
...
snippet i "if for error check" "is_err_declared_in_prev_statement()" bx
```